ref/general/authentication.adoc

```diff
@@ -34,18 +38,30 @@ Open Liberty supports the following options for SSO authentication:
 - link:/docs/ref/feature/#socialLogin-1.0.html[Social Media Login] +
 Users can authenticate to websites that are hosted on the Open Liberty server by logging in with their social media accounts. When the `socialLogin-1.0` feature is enabled, login with Facebook, Twitter, Google, Linkedin, and GitHub is automatically configured. You can also define your own configuration for any social media platform that is based on the OAuth 2.0 or OpenID Connect 1.0 standards.
 
+- link:/docs/ref/general/#/docs/concept/sso-config-json.html[JSON Web Tokens] +
+JSON Web Tokens (JWT) are used to propagate the user identity that is established by Single sign-on (SSO) between different microservice applications. By configuring SSO user authentication with JWT, you can manage the identity that is shared between applications.
```

Comment:
I don't think this is right. It isn't describing what a JWT is, it is describing a usage pattern as if that is what it is. JWT is a way to represent an authenticated user identity, it can then be used between microservices. I don't think SSO is necessarily relevant.

Reply:
should I remove this section altogether from the list of SSO options? It's based on this doc draft for the Jwt-1.0 feature : https://draft-openlibertyio.mybluemix.net/docs/ref/general/#/docs/concept/sso-config-json.html per #901

Reply:
No, I think we should have a better description of what JWT is, rather than stating what it is used for. That should probably find its way into any description elsewhere of JWT.

Reply:
@NottyCode new commit adds a description of what a JWT is..